README.md

```diff
@@ -99,6 +99,7 @@ and health-checking mechanisms.
     cd cortx-motr
 
     scripts/m0 make rpms
+    rm ~/rpmbuild/RPMS/x86_64/cortx-motr-test*.rpm
```

Comment:
I feel that it may not be correct to delete motr-test rpm just for proper installation.
If we delete motr test rpm (without installing it), we may not be able to run motr-ut I guess.

Your views please @mssawant , @knekrasov , @madhavemuri , @yeshpal-jain-seagate

Reply:
@suvratjoshi, @vaibhavparatwar  The win-win solution for this is that we **explicitly** put each RPM names to be installed. 
Instead of `sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-*.rpm`
it should be:
```sh
sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-2*.rpm
sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-ivt-*.rpm
sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-debuginfo-*.rpm
sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-devel-*.rpm
```
What do you guys think? I will update the PR if this approach is good enough.

Reply:
@daniarherikurniawan , why are we deleting the rpms in the first place?

Reply:
Hi @johnbent , the only RPM that shouldn't be installed is `cortx-motr-tests-ut-2.0.0-*.el7.x86_64.rpm`. Somehow that RPM will cause an error during installation and prevent the installation of all other RPM. The error is:
```sh
error: Failed dependencies:
	cortx-motr conflicts with cortx-motr-tests-ut-2.0.0-1_gite28e4d3_3.10.0_1127.19.1.el7.x86_64
```
Therefore, one of the solutions is to delete it (the cortx-motr-tests-*.rpm), but we have a better solution on another PR that avoids the deletiion.